scripts/get_version.py

```diff
@@ -28,6 +28,23 @@
 TIMEOUT = 3
 
 
+def fetch_pypi_versions(distribution: str) -> list[str]:
+    url = URL_TEMPLATE.format(PREFIX + distribution)
+    retry_strategy = Retry(
+        total=RETRIES, status_forcelist=RETRY_ON
+    )
+    with requests.Session() as session:
+        session.mount("https://", HTTPAdapter(max_retries=retry_strategy))
+        resp = session.get(url, timeout=TIMEOUT)
+    if not resp.ok:
+        if resp.status_code == 404:
+            # Looks like this is first time this package is ever uploaded.
+            return []
+        raise ValueError("Error while retrieving version")
+    data = resp.json()
+    return data["releases"].keys()
```

Comment:
This does not return `list[str]`. You can change the annotation to `Iterable` or change this line to `return list(data["releases"].keys())`.

Reply:
Good catch. We should start to type check these scripts at some point.

Reply:
They are already type checked, but `resp.json()` returns `Any`.